Project1/ExtendedCalculator.cs

```diff
@@ -8,5 +8,17 @@ namespace Project1
 {
     internal class ExtendedCalculator
     {
+        public bool PrimeNumber(int num1)
```

Comment:
W nazwie dajmy słówko `Get` czyli `GetPrimeNumber`

Reply:
zmienna `num1` też słaba jest, jak już potrzebna jedna cyfra to nazwij to `number`